ckanext/stadtzhharvest/harvester.py

```diff
@@ -379,6 +407,21 @@ def _get_existing_package(self, package_dict):
             log.debug('Could not find pkg %s' % package_dict['name'])
         return existing_package
 
+    def _get_existing_packages_names(self, harvest_job):
+        context = self._create_new_context()
+        try:
+            existing_packages = get_action('package_search')(
+                context,
+                {'fq': 'harvest_source_id:"{0}"'.format(harvest_job.source_id)}
+            )
+            existing_packages_nameids = \
```

Comment:
Can you refactor the variable-names that contain `..._nameids` to `_names`? In CKAN they are called names even though they are used as IDs. We want the code to be consistent so let's stick with the CKAN-definition and not invent something else.

Reply:
Done.